neofetch

```diff
@@ -965,35 +965,51 @@ get_cpu_usage() {
 get_gpu() {
     case "$os" in
         "Linux")
-            gpu="$(lspci -mm | awk -F '\\"|\\" \\"' '/"Display|"3D|"VGA/ {print $3 " " $4}')"
-
-            case "$gpu" in
-                *"advanced"*)
-                    gpu="${gpu//Intel*$'\n'}"
-                    gpu="${gpu/'[AMD/ATI]' }"
-                    gpu="${gpu/'[AMD]' }"
-                    gpu="${gpu/OEM }"
-                    gpu="${gpu/Advanced Micro Devices, Inc.}"
-                    gpu="${gpu/ \/ *}"
-                    gpu="${gpu/*\[}"
-                    gpu="${gpu/\]*}"
-                    gpu="AMD $gpu"
-                ;;
+            IFS=$'\n'
+            gpus=($(lspci -mm | awk -F '\\"|\\" \\"' '/"Display|"3D|"VGA/ {print $3 " " $4}'))
+
+            # Number the GPUs if more than one exists.
+            (( "${#gpus[@]}" > 1 )) && gpu_num=0
+
+            for gpu in "${gpus[@]}"; do
+                case "$gpu" in
+                    *"advanced"*)
+                        gpu="${gpu/'[AMD/ATI]' }"
+                        gpu="${gpu/'[AMD]' }"
+                        gpu="${gpu/OEM }"
+                        gpu="${gpu/Advanced Micro Devices, Inc.}"
+                        gpu="${gpu/ \/ *}"
+                        gpu="${gpu/*\[}"
+                        gpu="${gpu/\]*}"
+                        gpu="AMD $gpu"
+                    ;;
 
-                *"nvidia"*)
-                    gpu="${gpu//Intel*$'\n'}"
-                    gpu="${gpu/*\[}"
-                    gpu="${gpu/\]*}"
-                    gpu="NVIDIA $gpu"
-                ;;
+                    *"nvidia"*)
+                        gpu="${gpu/*\[}"
+                        gpu="${gpu/\]*}"
+                        gpu="NVIDIA $gpu"
+                    ;;
 
-                *"virtualbox"*)
-                    gpu="VirtualBox Graphics Adapter"
-                ;;
-            esac
+                    *"intel"*)
+                        gpu="Intel Integrated Graphics"
+                    ;;
 
-            [[ "$gpu" == *"intel"* ]] && \
-                gpu="Intel Integrated Graphics"
+                    *"virtualbox"*)
+                        gpu="VirtualBox Graphics Adapter"
+                    ;;
+                esac
+
+                if [[ "$gpu_brand" == "off" ]]; then
+                    gpu="${gpu/AMD }"
+                    gpu="${gpu/NVIDIA }"
+                    gpu="${gpu/Intel }"
+                fi
+
+                prin "GPU${gpu_num}" "$gpu"
```

Comment:
You may want to change the `GPU` one into `${subtitle}`, like the battery function.